credentials/tls.go

```diff
@@ -44,10 +44,27 @@ func (t TLSInfo) AuthType() string {
 	return "tls"
 }
 
+// cipherSuiteLookup returns the string version of a TLS cipher suite ID.
+func cipherSuiteLookup(cipherSuiteID uint16) string {
+	suites := tls.CipherSuites()
+	insecureSuites := tls.InsecureCipherSuites()
+	for i := range suites {
```

Comment:
Please remove the local var and use the value part of the range iterator:

```go
for _, s := range tls.CipherSuites() {
	if s.ID == cipherSuiteID {
		return s.Name
	}
}
<etc>
```

Reply:
This was actually my original approach, but I worried that I was doing a struct copy on each iteration. I'll make the change.

Reply:
That should be okay..  If we want to optimize this, we should just build the map we used to have using these functions.  (I'm OK either way.)